imports/startup/client/validator.js

```diff
@@ -55,6 +55,22 @@ isEmail = function(value) {
   return false;
 };
 
+/**
+ * Validates that the admin has set email server, it should be a proper email address
+ */
+
+isAdminEmailServerOn = function(value) {
+  const requiredUsername = Meteor.settings.public.smtp.username;
+  const requiredServer = Meteor.settings.public.smtp.server;
+  const requiredPassword = Meteor.settings.public.smtp.password;
+  if (requiredUsername.includes(mailgun.org)&&requiredServer.includes(mailgun.org)&&requiredPassword.length>0) {
```

Comment:
I think you are missing a few quotes in this if condition.

Reply:
Thank you for this review, I have added the quotes and changed the comment above for the function.